task_pool.py better exception handling

Smackteo opened the pull request:
#### 50daba6f8f915913eca6fda2ec5666b83de3f8d1
<pre>
task_pool.py better exception handling
<a href="https://bugs.webkit.org/show_bug.cgi?id=243952">https://bugs.webkit.org/show_bug.cgi?id=243952</a>

Reviewed by Jonathan Bedard.

* Tools/Scripts/libraries/webkitcorepy/setup.py:
* Tools/Scripts/libraries/webkitcorepy/webkitcorepy/__init__.py:
* Tools/Scripts/libraries/webkitcorepy/webkitcorepy/task_pool.py:
(_BiDirectionalQueue.receive):

Canonical link: <a href="https://commits.webkit.org/253499@main">https://commits.webkit.org/253499@main</a>
</pre>

JonWBedard reviewed (approved):
Please bump the minor version in `Tools/Scripts/libraries/webkitcorepy/setup.py` and `Tools/Scripts/libraries/webkitcorepy/webkitcorepy/__init__.py`

Smackteo committed:
task_pool.py better exception handling
https://bugs.webkit.org/show_bug.cgi?id=243952

Reviewed by Jonathan Bedard.

* Tools/Scripts/libraries/webkitcorepy/setup.py:
* Tools/Scripts/libraries/webkitcorepy/webkitcorepy/__init__.py:
* Tools/Scripts/libraries/webkitcorepy/webkitcorepy/task_pool.py:
(_BiDirectionalQueue.receive):

Canonical link: https://commits.webkit.org/253499@main
